### PR TITLE
Improve Readme.

### PR DESCRIPTION
**Pull request by afiebig**
Also, fix some visual bugs in the readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,73 +1,110 @@
...
 
-## First run
+### First run
 
 Note: Only tested on Mac OS X and Linux so far.
 
 1. If you are using voltos, the firs step is to run `voltos use gobdigital-interoperabilidad` inside the project root folder.
 
-- To do things more simple, you can create a function inside your `.bashrc` or `.bashrc.local` and replace make with vmake in every command.
+    - To do things more simple, you can create a function inside your `.bashrc` or `.bashrc.local` and replace make with vmake in every command.
```
Comment (eugenioLopezRamos) [inline README.md]: To make things simpler en vez de To do things more simple

File: README.md [hunk 2]
```diff
@@ -1,73 +1,110 @@
...
 
-- `SECRET_KEY_BASE`: A random string that can be generated via `rails secret`. It should be the *same* for *every* instance running in production.
+- `SECRET_KEY_BASE`: A random string that can be generated via `rails secret`. It should be the **same** for **every** instance running in production.
 
 - `DATABASE_URL`: A pointer to the database (e.g: `postgres://myuser:mypass@localhost/somedatabase`).
 
-- `REDIS_URL`: A pointer to the Redis instance (e.g: `redis://myuser:mypass@redis-host:6379`)
+- `REDIS_URL`: A pointer to the Redis instance (e.g: `redis://myuser:mypass@redis-host:6379`).
+
+- `ISSUER_OIDC`: Is the URL for te issuer of ClaveUnica OpenID.
```
Comment (eugenioLopezRamos) [inline README.md]: the issuer

File: README.md [hunk 3]
```diff
@@ -107,13 +144,17 @@ Production should run the latest [`egob/interoperabilidad`](https://hub.docker.c
 
 - `AGREEMENT_CLIENT_TOKEN_EXPIRATION_IN_SECONDS`: Time to live for auth client tokens given to consumers after an agreement is signed to use protected services.
 
-- `TRACEABILITY_ENDPOINT`: Address of the traceability dashboard
+- `TRACEABILITY_ENDPOINT`: Address of the traceability dashboard.
+
+- `TRAZABILIDAD_SECRET`: This secret is shared with `Plataforma de Interacciones` to check that it can access te endpoit containing trazability info.
```
Comment (eugenioLopezRamos) [inline README.md]: "the endpoint containing traceability info"

File: README.md [hunk 4]
```diff
@@ -180,62 +220,64 @@ The web containers will enqueue background jobs into a queue stored in redis. In
         egob/interoperabilidad \
         bundle exec sidekiq
 
-### Run the web containers
+#### Run the web containers
 
-Don't forget to run the docker image with all the env variablaes and without special command in one or more machines and point a load balancer to it :).
+Don't forget to run the docker image with all the env variablaes and without special command in one or more machines and point a load balancer to it :smiley: .
```
Comment (eugenioLopezRamos) [inline README.md]: variables en vez de variablaes

File: README.md (hunk 1 above)
Comment (afiebig) [reply README.md]: I only changes the indentation level here :(

File: README.md (hunk 4 above)
Comment (afiebig) [reply README.md]: again, just put a smiley here. 😭

File: README.md [hunk 5]
```diff
@@ -1,73 +1,106 @@
...
 
-## First run
+### First run
 
 Note: Only tested on Mac OS X and Linux so far.
 
 1. If you are using voltos, the firs step is to run `voltos use gobdigital-interoperabilidad` inside the project root folder.
 
-- To do things more simple, you can create a function inside your `.bashrc` or `.bashrc.local` and replace make with vmake in every command.
+    - To do things simpler, you can create a function inside your `.bashrc` or `.bashrc.local` and replace make with vmake in every command.
```
Comment (eugenioLopezRamos) [inline README.md]: To make things simpler

File: README.md [hunk 6]
```diff
@@ -107,13 +140,17 @@ Production should run the latest [`egob/interoperabilidad`](https://hub.docker.c
 
 - `AGREEMENT_CLIENT_TOKEN_EXPIRATION_IN_SECONDS`: Time to live for auth client tokens given to consumers after an agreement is signed to use protected services.
 
-- `TRACEABILITY_ENDPOINT`: Address of the traceability dashboard
+- `TRACEABILITY_ENDPOINT`: Address of the traceability dashboard.
+
+- `TRAZABILIDAD_SECRET`: This secret is shared with `Plataforma de Interacciones` to check that it can access te endpoit containing traceability info.
```
Comment (eugenioLopezRamos) [inline README.md]: endpoint en vez de endpoit

**Pull request merged**